### DISCUSSION TITLE
Contacts list is too long when we have a lot of contacts

### DISCUSSION THREAD

**Discussion by yankaiqian** [nioc/xmpp-web#110] [Ideas]
Is it possible to add a button to the Contacts and we can click it to make the contacts list expanded or collapsed ?

The list is too long when we have a lot of contacts, it will be better if the Contacts can be collapsed like the GROUP item.

**Comment by nioc** [comment]
Hello, this is an interesting feedback 🙂

I prefer not adding another level of collapsable, but may be adding some filters like:
- active users
- unread conversations
- search by name

Does it cover your need?

**Reply by nioc** [reply]
Another (simpler) way to handle it: add a fixed max height to the contacts list, then using an automatic scrollbar for browsing contacts.

**Reply by yankaiqian** [reply]
Yes, either filter option or fixed height is fine, it would be better if we can have both, thank you!
